pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -258,6 +262,16 @@
     // Name of load manager to use
     @FieldContext(dynamic = true)
     private String loadManagerClassName = "com.yahoo.pulsar.broker.loadbalance.impl.SimpleLoadManagerImpl";
+    @FieldContext(dynamic = true)
+    private String activeVersion = "";
```

Comment:
If `activeVersion` is configurable on each broker, different broker could have different views on what's the last version. 

Wouldn't it be easier to keep it that the last version is the active version?